app/src/main/res/values/bool.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<resources>
+    <bool name="text_caps_logout_btn">false</bool>
+    <bool name="photo_adjust_view_bounds">true</bool>
+    <bool name="edit_btn_font_padding">false</bool>
+    <bool name="text_caps_edit_btn">false</bool>
+    <bool name="contacts_btn_font_padding">false</bool>
+    <bool name="text_caps_contacts_btn">true</bool>
+</resources>
```

Comment:
never stored booleans in the separate folder, it is not necessary. The point of moving something to the resources folder is to support different configurations with ease, such as different screen orientations or day/night theme

Reply:
I have fixed it